.devconteiner/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "CDE for robotology-superbuild training",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"forwardPorts": [6080],
+	"remoteUser": "codespace",
+    "postAttachCommand": {
+        "vnc": "start-vnc-session.sh",
+        "download_bashrc": "cd ${HOME} && wget -O .bashrc_iCub https://raw.githubusercontent.com/icub-tech-iit/icub-os-files/master/user-environment/bashrc_iCub_superbuild",
+        "append_bashrc": "cd ${HOME} && printf \"if [ -f ~/.bashrc_iCub ]; then\n . ~/.bashrc_iCub\nfi\" | cat - ~/.bashrc > temp && mv temp ~/.bashrc && source ~/.bashrc && cd -",
+        "set_obj_subdir": "cd ~ && printf \"export OBJ_SUBDIR=build\n\" | cat - ~/.bashrc > temp && mv temp ~/.bashrc && source ~/.bashrc && cd -"
+    }
```

Comment:
This is likely not correct as these commands need to be executed in a row, I suppose.
Instead, here each line runs into an independent terminal.

I'm afraid that it was the same in Gitpod so there was already a problem in the first place.

You could better churn out a `setup.sh` bash script (make it executable and put it under .devcontainer folder). Then, you can refactor it this way:

```json
"vnc": "start-vnc-session.sh",
"setup": "setup.sh"
```

Reply:
Alternatively, you may embed those operations inside the Dockerfile.